docs/2gp_testing.rst

```diff
@@ -0,0 +1,123 @@
+Testing with Second-Generation Packaging
+========================================
+
+CumulusCI and the CumulusCI Flow make it easy to harness the power of second-generation
+managed packages to implement an advanced, comprehensive testing process for *both* 
+first- and second-generation managed package products.
+
+This process yields multiple benefits:
+
+* You can test managed packages before merging code.
+* You gain the ability to perform end-to-end testing for packages with more complex
+  dependency structures earlier in the development lifecycle.
```

Comment:
Would it be clearer to simply say "with multiple packages" rather than "for packages with more complex dependency structures"?